lib/index.js

```diff
@@ -47,10 +47,10 @@
   var makeApiCall = require('./internal/make-api-call').inject(options || {});
 
   var makeApiMethod = function(apiConfig) {
-    return function(query, callback) {
+    return function(query, callback, customParams) {
       query = apiConfig.validator(query);
       query.supportsClientId = apiConfig.supportsClientId !== false;
-      return makeApiCall(apiConfig.url, query, callback);
+      return makeApiCall(apiConfig.url, query, callback, customParams);
```

Comment:
If you copy the customParams into the query here (after the validator has run) would that mean you don't need to make any changes to make-api-call.js ?